views/address.tmpl

```diff
@@ -5,6 +5,8 @@
 {{template "html-head" printf "Decred Address %s" .Data.Address}}
 <body class="{{ theme }}">
     {{template "navbar" . }}
+    {{$TxTime := .OldestTxTime}}
+    {{$IsLite := .IsLiteMode}}
```

Comment:
`{{with .Data}}` (line 10) changes our scope and thats why this variables have to be declared outside so that they can be used in the scope of `{{with .Data}}` without its effect.

Reply:
Yeah, that's OK.  Just note that we can do `$.IsLiteMode` to get back to the root scope regardless of the current scope.